325: Automatische Übernahme der Wohnfläche

joergreichert opened the pull request:
erweitern 

Task-Url: http://polymap.org/kaps/ticket/325

Summe Wohnflächen aus Ertragswertverfahren beziehen

stundzig commented on plugins/org.polymap.kaps/src/org/polymap/kaps/ui/form/VertragsdatenBaulandSonstigesFormEditorPage.java:
```diff
@@ -101,6 +98,32 @@ public void createFormContent( final IFormEditorPageSite site ) {
+        	);
+        }
+        if(gesamtFlaeche != null) {
+        	vb.wohnflaeche().set(gesamtFlaeche);
```
Ziehe diese Berechnung bitte mal in eine separate Methode, die du nach createFormContent aufrufst. Den Wert setze bitte nur im Form mit site.setFieldValue("wohnfläche") oderso ähnlich.

joergreichert committed:
368: Fehlermeldung bei Meldung STALA 

Task-Url: http://polymap.org/kaps/ticket/368

Berechnung am Ende des create Forms und setzt nur den im Feld, nicht den
im Modell (der wird erst beim Speichern propagiert)
